app/Models/Lookup/Department.php

```diff
@@ -19,37 +20,38 @@
  *
  * @property \Jenssegers\Date\Date $created_at
  * @property \Jenssegers\Date\Date $updated_at
+ *
  * @property \Illuminate\Database\Eloquent\Collection $managers
  * @property \Illuminate\Database\Eloquent\Collection $job_posters
+ *
+ * Localized Properties:
+ * @property string $value
  */
 class Department extends BaseModel
 {
-
     use CrudTrait;
     use HasTranslations;
 
     /**
-     * @var $fillable string[]
-     */
-    protected $fillable = [
-        'id',
-        'name',
```

Comment:
name and impact need to remain fillable, in order to be editable on the admin portal.